src/vivarium_cluster_tools/runner.py

```diff
@@ -44,9 +45,7 @@ def uge_specification(peak_memory, project, job_name):
         if os.environ['SGE_CLUSTER_NAME'] == "dev":
             project = None
         else:  # prod or new cluster
-            assert project in ['proj_cost_effect', 'proj_cost_effect_diarrhea',
-                               'proj_cost_effect_dcpn', 'proj_cost_effect_conic',
-                               'proj_csu'], 'Script only for use by sanofi and cost effectiveness teams.'
+            assert project in CLUSTER_PROJECTS, 'Script only for use by sanofi and cost effectiveness teams.'
```

Comment:
Can we tell the user what valid projects there are? This wouldn't be a super helpful error message